schema/models.py

```diff
@@ -367,4 +367,14 @@ class FlowModel(models.Model):
  
 class CredentialsModel(models.Model):
     id = models.ForeignKey(AUTH_USER_MODEL, primary_key=True, on_delete=models.CASCADE)
-    credential = CredentialsField()
\ No newline at end of file
+    credential = CredentialsField()
+
+
+class Test_model(models.Model):
```

Comment:
what is this model for? Can you put comments what this is for? And maybe change the model name to be more descriptive